FirebaseTestProject/ViewModels/SignInViewModel.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  SignInViewModel.swift
+//  SignInViewModel
+//
+//  Created by Tim Miller on 8/12/21.
+//
+
+
+import FirebaseAuth
+
+class SignInViewModel: ObservableObject {
+    
+    let auth = Auth.auth()
+    let userDefault = UserDefaults.standard
```

Comment:
This probably doesn't need to be its own variable

Reply:
removed